store/actions/test.js

```diff
@@ -0,0 +1,32 @@
+/** @format */
+
+import * as actions from './';
+import ActionTypes from './ActionTypes';
+
+describe( 'Store', () => {
+	describe( 'actions', () => {
+		it( 'should create an action to focus a block', () => {
+			const action = actions.focusBlockAction( 1 );
+			expect( action.type ).toEqual( ActionTypes.BLOCK.FOCUS );
+			expect( action.rowId ).toEqual( 1 );
+		} );
+
+		it( 'should create an action to move block up', () => {
+			const action = actions.moveBlockUpAction( 1 );
+			expect( action.type ).toEqual( ActionTypes.BLOCK.MOVE_UP );
+			expect( action.rowId ).toEqual( 1 );
+		} );
+
+		it( 'should create an action to move block down', () => {
+			const action = actions.moveBlockDownAction( 1 );
+			expect( action.type ).toEqual( ActionTypes.BLOCK.MOVE_DOWN );
+			expect( action.rowId ).toEqual( 1 );
+		} );
+
+		it( 'should create an action to delete a block', () => {
+			const action = actions.deleteBlockAction( 1 );
+			expect( action.type ).toEqual( ActionTypes.BLOCK.DELETE );
```

Comment:
We could add `expect( action.type ).toBeDefined();` here to prevent previous problem

Reply:
Makes sense, added with `91f2180`.